packages/create-sitecore-jss/src/templates/angular-xmcloud/src/app/components/image/image.component.html

```diff
@@ -0,0 +1,36 @@
+<ng-container [ngTemplateOutlet]="variant"></ng-container>
+
+<ng-template #default>
+  <div class="component image {{ styles }}" [attr.id]="id" *ngIf="rendering.fields; else empty">
```

Comment:
we can use the `ngClass` directive to improve readability.
Similarly wherever classes are defined.

Reply:
yea, makes sense, I will update